tests/acceptance/features/bootstrap/ShareesContext.php

```diff
@@ -202,6 +188,32 @@ public function getArrayOfShareesResponded(
 		return $sharees;
 	}
 
+	/**
+	 * @param string $user
+	 * @param TableNode $body
+	 *
+	 * @return void
+	 */
+	public function getShareesWithParameters(string $user, TableNode $body):void {
+		$user = $this->featureContext->getActualUsername($user);
+		$url = '/apps/files_sharing/api/v1/sharees';
+		$this->featureContext->verifyTableNodeColumnsCount($body, 2);
+		$parameters = [];
+		foreach ($body->getRowsHash() as $key => $value) {
+			$parameters[] = "$key=$value";
+		}
+		if (!empty($parameters)) {
+			$url .= '?' . \implode('&', $parameters);
+		}
+
+		$response = $this->ocsContext->sendRequestToOcsEndpoint(
+			$user,
+			'GET',
+			$url
+		);
+		$this->featureContext->setResponse($response);
```

Comment:
we MUST NOT use `setResponse` in helper methods. Please, use it in the step def methods only
```suggestion
		return $this->ocsContext->sendRequestToOcsEndpoint(
			$user,
			'GET',
			$url
		);
```